core/websocket_client.py

```
@@ -1,48 +1,94 @@
+                    net_cost = slippage + fee + impact
+
+                    # maker_taker_probs = predict_maker_taker_proba(maker_taker_model, usd_amount, spread, depth)
+
+                    if maker_taker_model:
+                        maker_taker_probs = predict_maker_taker_proba(maker_taker_model, usd_amount, spread, depth)
+                        maker_str = f"""
+                        Maker Prob:         {maker_taker_probs['maker']:.2%}
+                        Taker Prob:         {maker_taker_probs['taker']:.2%}
+                        """
+                    else:
+                        maker_str = "Maker/Taker prediction: Not available (model not trained)\n"
```

Ruff (0.11.9)

Local variable `maker_str` is assigned to but never used

Remove assignment to unused variable `maker_str`

(F841)